src/types/predicates.go

```diff
@@ -37,6 +37,29 @@ func IsAlias(s string) bool {
 	return has
 }
 
+func IsTypeNullable(typ Map) bool {
+	isNullable := false
+	if typ.m == nil {
+		return true // We consider that if the type is not inferred, then it is mixed
+	}
+	typ.Iterate(func(t string) {
+		if t == "null" || strings.Contains(t, "mixed") {
+			isNullable = true
+		}
+	})
+	return isNullable
+}
+
+func IsTypeMixed(typ Map) bool {
```

Comment:
why not just `type.Contains("mixed")`? in this case, you're looking at `[]mixed` simply as `mixed`

Reply:
You are right. I missed this